umbraco/Umbraco-CMS
# V14: Remove old backoffice project.

# Notes
- Removes the `Umbraco.Web.UI` project
- Removes the `Umbraco.Web.UI.Client` project
- Removes the `Umbraco.Web.Backoffice` project
- Moves a ton of the old DI logic into management API
- Moves the authentication logic into the management API.

# How to test
- The new backoffice should still run
- Poke around in the new backoffice. Can you still create/Publish/delete content, contenttypes, etc.
- We should test External login at some point to make sure this pr doesn't ruin it.
